app-creator/README.md

```diff
@@ -0,0 +1,81 @@
+# Earth Engine App Creator
+
+## Setup
+
+Install dependencies:
+
+```bash
+npm i
+```
+
+## Build
+
+This sample uses the TypeScript compiler to produce JavaScript that runs in modern browsers.
+
+To build the JavaScript version of your component:
+
+```bash
+npm run build
+```
+
+To watch files and rebuild when the files are modified, run the following command in a separate shell:
+
+```bash
+npm run build:watch
+```
+
+## Testing
+
+This repository uses Karma, Chai, Mocha, and the open-wc test helpers for testing. See the [open-wc testing documentation](https://open-wc.org/testing/testing.html) for more information.
```

Comment:
Would you be willing to convert the Chai framework to use Jasmine instead? https://jasmine.github.io/2.0/introduction.html

Looking at the test files, they are almost identical syntactically and it's more standard in our OS repos:
https://github.com/google/earthengine-api/tree/master/javascript/test

If you don't want to convert it in this PR, you can do it in a follow up.

Reply:
Cool, I'll do it in a follow up :)